spring-cloud-gcp-samples/spring-cloud-gcp-vision-ocr-sample/README.adoc

```diff
@@ -0,0 +1,53 @@
+= Spring GCP Cloud Vision OCR Sample Application
+
+This demonstrates how to run Google Cloud Vision https://cloud.google.com/vision/docs/ocr[Optical Character Recognition (OCR)] analysis on PDF and TIFF documents.
+OCR is a process which allows you to detect and extract the text from digital documents.
+
+This sample application allows you to upload documents to a specified https://cloud.google.com/storage/docs/key-terms#buckets[Google Cloud Storage bucket] and run OCR on those documents you uploaded.
+After your document is uploaded and processed, the application will allow you to access the extracted text of each document.
+
+== Running the sample
+
+This section describes how to run the sample and use it to extract text from some documents.
+
+=== 1. Setting up your GCP Project
+
+a. Create a Google Cloud Platform project with https://cloud.google.com/billing/docs/how-to/modify-project#enable-billing[billing enabled], if you don't have one already.
+
+b. Enable the Google Cloud Vision API from the "APIs & Services" menu of the Google Cloud Console.
```

Comment:
I believe you can also enable it via a gcloud command. Can you provide it here as well? Thanks!